schema-conformance/action.yaml

```diff
@@ -89,9 +89,11 @@ runs:
     - id: map-outputs
       if: always()
       run: |
-        echo "::set-output name=success::${{ fromJSON(steps.publish-junit.outputs.json).stats.tests_succ }}"
-        echo "::set-output name=failed::${{ fromJSON(steps.publish-junit.outputs.json).stats.tests_fail }}"
-        echo "::set-output name=error::${{ fromJSON(steps.publish-junit.outputs.json).stats.tests_error }}"
-        echo "::set-output name=skip::${{ fromJSON(steps.publish-junit.outputs.json).stats.tests_skip }}"
-        echo "::set-output name=total::${{ fromJSON(steps.publish-junit.outputs.json).stats.tests }}"
+        echo "${{ steps.publish-junit.outputs.json }}"
+        cat $PWD/reports/junit.xml
+        echo "success=${{ fromJSON(steps.publish-junit.outputs.json).stats.tests_succ }}" >> $GITHUB_OUTPUT
```

Comment:
How about writing all these to a file at once and piping that file output to GH_OUTPUT? we can also cat that file in debug mode and easily be able to debug if there are any syntax issues or something.